infra/modules/azuread-grant-all-users/main.tf

```diff
@@ -10,25 +10,32 @@ terraform {
   }
 }
 
-data "azuread_application_published_app_ids" "well_known" {}
+# TODO: if grant can be made fully through API, do it here; until then, TODO file is best option
```

Comment:
True, that command seems to work only for delegated permissions...

To perform an [admin consent](https://docs.microsoft.com/en-us/cli/azure/ad/app/permission?view=azure-cli-latest#az-ad-app-permission-admin-consent):

```
az ad app permission admin-consent --id {appId}
```

To add a grant in the AAD application:

```
az ad app permission grant --id {appId} --api {permissionId} 
```

All permissions we have configured are application are *application* kind, grant admin will be the right one for these

Reply:
weird thing is that it didn't give errors; the grant just had no effect